server.js

```diff
@@ -63,8 +63,8 @@
 const http = require('http');
 const mongoose = require('mongoose');

```

Comment:
Consider using a descriptive variable name instead of 'HOSTNAME'. This can improve code readability and maintainability.